flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -0,0 +1,76 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TYPE organization_status_type AS ENUM ('active', 'suspend', 'deleted');
+CREATE TYPE user_status_type AS ENUM ('active', 'deactivated', 'deleted');
+CREATE TYPE gender_type AS ENUM ('male', 'female');
+
+CREATE TABLE IF NOT EXISTS sso_organization (
+    organization_id SERIAL PRIMARY KEY NOT NULL,
+    organization_name VARCHAR(255) NOT NULL,
+    status organization_status_type,
+    address VARCHAR(255),
+    contact_number VARCHAR(255),
+    email VARCHAR(255),
+    country VARCHAR(255),
+    state VARCHAR(255),
+    city VARCHAR(255),
+    postal_code VARCHAR(255),
+    website_url VARCHAR(255),
+    kvp JSONB,
+    date_created TIMESTAMPTZ DEFAULT now(),
+    date_modified TIMESTAMPTZ DEFAULT now(),
+);
+
+CREATE TABLE IF NOT EXISTS sso_role (
+    role_id UUID DEFAULT uuid_generate_v4() PRIMARY KEY NOT NULL,
+    role_name VARCHAR(255) NOT NULL,
+    role_description VARCHAR(255),
+    organization_id INT NOT NULL,
+    FOREIGN KEY (organization_id) REFERENCES sso_organization(organization_id)
+);
+
+CREATE TABLE IF NOT EXISTS sso_user (
+    user_id UUID DEFAULT uuid_generate_v4() PRIMARY KEY NOT NULL,
+    username VARCHAR(255) NOT NULL,
+    password VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL,
+    kvp JSONB,
+    date_created TIMESTAMPTZ DEFAULT now(),
+    date_modified TIMESTAMPTZ DEFAULT now(),
+    modified_by BIGINT,
```

Comment:
Bởi vì nó thường chính là updated by `user_id` 
```suggestion
    modified_by UUID,
```